frontend/probloom/src/components/ProblemSet/ProblemSetDetail/CommentComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const CommentCompoent = (props) => {
```

Comment:
This typo was detected by SonarCloud. Please rename it to `CommentComponent`.

Reply:
Oh, I'll fix it.